its/org.sonarlint.eclipse.its/src/org/sonarlint/eclipse/its/RulesConfigurationTest.java

```diff
@@ -55,50 +58,85 @@ public void deactivate_rule() throws Exception {
-      tuple("/java-exclude-rules/src/hello/Hello3.java", 12, "Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed."));
+    checkIssueIsDefault(project);
 
     new JavaPackageExplorerBot(bot).expandAndSelect("java-exclude-rules", "src", "hello", "Hello3.java");
 
     onTheFly.bot().tree().select(1).contextMenu("Deactivate rule").click();
 
     JobHelpers.waitForJobsToComplete(bot);
-    markers = Arrays.asList(project.findMember("src/hello/Hello3.java").findMarkers(MARKER_ON_THE_FLY_ID, true, IResource.DEPTH_ONE));
+    List<IMarker> markers = Arrays.asList(project.findMember("src/hello/Hello3.java").findMarkers(MARKER_ON_THE_FLY_ID, true, IResource.DEPTH_ONE));
     assertThat(markers).extracting(markerAttributes(IMarker.LINE_NUMBER, IMarker.MESSAGE)).containsOnly(
       tuple("/java-exclude-rules/src/hello/Hello3.java", 9, "Replace this use of System.out or System.err by a logger."));
 
     reactivateRuleUsingUI();
 
     JobHelpers.waitForJobsToComplete(bot);
-    markers = Arrays.asList(project.findMember("src/hello/Hello3.java").findMarkers(MARKER_ON_THE_FLY_ID, true, IResource.DEPTH_ONE));
-    assertThat(markers).extracting(markerAttributes(IMarker.LINE_NUMBER, IMarker.MESSAGE)).containsOnly(
-      tuple("/java-exclude-rules/src/hello/Hello3.java", 9, "Replace this use of System.out or System.err by a logger."),
-      tuple("/java-exclude-rules/src/hello/Hello3.java", 12, "Refactor this method to reduce its Cognitive Complexity from 24 to the 15 allowed."));
+    checkIssueIsDefault(project);
+
+    changeRuleParamValue(project);
 
     openComplexityRule();
-    bot.spinner().setSelection(10);
+
+    bot.link().click();
     bot.button("Apply").click();
     bot.activeShell().close();
+
     JobHelpers.waitForJobsToComplete(bot);
-    markers = Arrays.asList(project.findMember("src/hello/Hello3.java").findMarkers(MARKER_ON_THE_FLY_ID, true, IResource.DEPTH_ONE));
-    assertThat(markers).extracting(markerAttributes(IMarker.LINE_NUMBER, IMarker.MESSAGE)).containsOnly(
-      tuple("/java-exclude-rules/src/hello/Hello3.java", 9, "Replace this use of System.out or System.err by a logger."),
-      tuple("/java-exclude-rules/src/hello/Hello3.java", 12, "Refactor this method to reduce its Cognitive Complexity from 24 to the 10 allowed."));
+    checkIssueIsDefault(project);
+  }
 
-    openComplexityRule();
+  @Test
+  public void ruleParametersGlobalDefaults() throws Exception {
+    SwtBotUtils.openPerspective(bot, JavaUI.ID_PERSPECTIVE);
+    IProject project = importEclipseProject("java/java-exclude-rules", "java-exclude-rules");
+    JobHelpers.waitForJobsToComplete(bot);
 
-    bot.link().click();
+    JavaPackageExplorerBot javaBot = new JavaPackageExplorerBot(bot);
+    javaBot.expandAndDoubleClick("java-exclude-rules", "src", "hello", "Hello3.java");
+    JobHelpers.waitForJobsToComplete(bot);
+    checkIssueIsDefault(project);
+    changeRuleParamValue(project);
+
```

Comment:
I would put assertion `checkIssueChanged` here for symetry with `checkIssueIsDefault`